### PR TITLE
Mark Hacktoberfest 2019 opportunities as expired

### LINKED ISSUE #468: Remove HacktoberFest related Swags
## Issue with an existing swag opportunity

- [x] I've checked that this isn't a duplicate issue.

Q. Which opportunity?  
A. : Many 

There are many Hacktoberfest related Swag opportunities which seem to have expired as HacktoberFest has ended. Why not remove all those.

### PR DESCRIPTION
**Pull request by manan2501**
Marked all hacktober swages as expired

- [ ] I've checked that this isn't a new swag opportunity proposal.
- [ ] I've checked that this isn't a duplicate pull request.

Fixes #468

### CODE REVIEW

**Comment by shakeabi** [comment]
@manan2501 Check what's the CI error, fix it and push it again

**Comment by manan2501** [comment]
> @manan2501 Check what's the CI error, fix it and push it again

It says `1) Microsoft has a valid reference` but I checked that reference is valid and works fine.

File: data.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
     "reference": "https://github.com/facebook/docusaurus/issues/1834",
     "image": "https://i.imgur.com/IbZCAWP.jpg",
     "dateAdded": "2019-10-27T12:37:23.000Z",
-    "tags": ["hacktoberfest", "stickers"]
+    "tags": ["hacktoberfest", "stickers", "expired"]
```
Comment (aslafy-z) [inline data.json]: This opportunity is not expired. They added a tee during the Hacktoberfest.

File: data.json [hunk 2]
```diff
@@ -294,7 +294,7 @@
     "reference": "https://blog.mayadata.io/mayadata-celebrates-hacktoberfest-2019",
     "image": "https://blog.mayadata.io/hs-fs/hubfs/Hacktoberfest-11-1.png",
     "dateAdded": "2019-10-02T16:48:14.000Z",
-    "tags": ["clothing", "device", "hacktoberfest", "stickers"]
+    "tags": ["clothing", "device", "hacktoberfest", "stickers", "expired"]
   },
   {
     "name": "Microsoft",
```
Comment (aslafy-z) [inline data.json]: This opportunity is expired

**Comment by KillMonger1** [comment]
I feel that the PR should be merged now, Its been a long time and the website is getting outdated.

**Pull request merged**